code/game/objects/items/RCD.dm

```diff
@@ -203,10 +203,23 @@ RLD
 	var/use_one_access = 0 //If the airlock should require ALL or only ONE of the listed accesses.
 	var/delay_mod = 1
 	var/canRturf = FALSE //Variable for R walls to deconstruct them
+	var/id
```

Comment:
not a descriptive var name

Reply:
I also don't see where you're actually applying it to the new conveyors.

Reply:
Noted. ID is the id assigned to the conveyor switch. I will make it more descriptive.

It is applied in line 'last_placed = new/obj/machinery/conveyor(A, cdir, id)'